plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/wizard/AbstractOpenShiftApplicationWizard.java

```diff
@@ -35,14 +36,18 @@
  * 
  * @author mlabuda@redhat.com
  */
-public abstract class NewOpenShiftApplicationWizard {
+public abstract class AbstractOpenShiftApplicationWizard {
 	
 	protected TreeViewerHandler treeViewerHandler = TreeViewerHandler.getInstance();
 	
 	protected String server;
 	protected String username;
 	
-	public NewOpenShiftApplicationWizard(String server, String username) {
+	public AbstractOpenShiftApplicationWizard(String server, String username) {
+		this(server, username, null);
+	}
+
+	public AbstractOpenShiftApplicationWizard(String server, String username, String project) {
```

Comment:
Did not you forget to do something with the project parameter? It's completely ignored in constructor body and this constructor is not used anywhere.